### PR TITLE
Corrected rendering of LaTeX

### PR DESCRIPTION
**Pull request by czgdp1807**
#### References to other Issues or PRs

#14203

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by czgdp1807** [comment]
ping @jksuom Can we merge this? Anything needed to be manually tested?

**Comment by oscarbenjamin** [comment]
What does this do? Can you reference the previous PR and/or issue in the OP?

**Comment by czgdp1807** [comment]
@oscarbenjamin Done. Please take a look.

File: doc/src/modules/parsing.rst [hunk 1]
```diff
@@ -58,19 +58,19 @@ don't need to be manually added by the user.
...
-`\LaTeX` parsing was ported from
+\LaTeX parsing was ported from
 `latex2sympy <https://github.com/augustt198/latex2sympy>`_. While functional
 and its API should remain stable, the parsing behavior or backend may change in
 future releases.
 
-`\LaTeX` Parsing Caveats
-------------------------
+\LaTeX Parsing Caveats
+---------------------------------
```
Comment (jksuom) [inline doc/src/modules/parsing.rst]: It this change accidental?

Comment (czgdp1807) [reply doc/src/modules/parsing.rst]: I don't know. Will build locally and will let you know.

Comment (czgdp1807) [reply doc/src/modules/parsing.rst]: I tried to build the docs and getting the following exception,
```python
PYTHONPATH=..: sphinx-build -b html -d _build/doctrees -W src _build/html
Running Sphinx v1.6.7

Extension error:
Unknown event name: config-inited
Makefile:49: recipe for target 'html' failed
make: *** [html] Error 1
```

Comment (sylee957) [reply doc/src/modules/parsing.rst]: It can be because of your sphinx version is old.
Can you update your sphinx version?

**Comment by czgdp1807** [comment]
With \LaTeX
render_latex

With `\LaTeX` (with back ticks)
render_latex_master

With `\mathrm{\LaTeX}`
render_mathrm

It looks like using the third alternative is a good solution. Let me know of your views. @jksuom @sylee957

File: doc/src/modules/parsing.rst (hunk 1 above)
Comment (czgdp1807) [reply doc/src/modules/parsing.rst]: Those extra `-` aren't making any difference. Will remove them in next commit.

**Comment by jksuom** [comment]
I think that `\LaTeX` was originally defined to be used in text mode (or paragraph and LR mode as Lamport says in C.3.1 of his book). However, it seems that `\LaTeX` was redefined many times for different fonts (see, e.g., `ltnews.cls`). So maybe `\mathrm{\LaTeX}` could also work though the result might depend on how `\mathrm` has been defined.

**Comment by asmeurer** [comment]
As noted in the issue, we should use `\mathrm{\LaTeX}`. Removing the math formatting here makes it look worse.

**Comment by asmeurer** [comment]
In fact the original pull request looks better in this regard, unless there are some formatting issues with it when it is built?

**Comment by czgdp1807** [comment]
I have restored the changes of the original PR. Let me know if any more changes are to be made.

**Comment by czgdp1807** [comment]
@asmeurer @jksuom The tests are passing, can we merge this?

**Comment by czgdp1807** [comment]
Any updates on this?

**Comment by jksuom** [comment]
I think that `\mathrm{\LaTeX}` is clumsy but, as `\LaTeX` does not seem to work, we should probably use it. After all, the logo is only rarely needed. But should we merge the original https://github.com/sympy/sympy/pull/14203, possibly with some modifications?

**Comment by czgdp1807** [comment]
Yes, we can do that. Actually I don't know how to push changes to someone else's PR.

**Comment by jksuom** [comment]
> I don't know how to push changes to someone else's PR. 

I'm not sure but, if "Allow edits from maintainers." was checked (as it probably was), you should be able to do that as explained [here](https://github.blog/2016-09-07-improving-collaboration-with-forks/).

**Comment by czgdp1807** [comment]
Closing in favour of the original PR.

**Pull request closed without merging**